content/pt/docs/concepts/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Conceitos do OpenTelemetry
+linkTitle: Conceitos
+description: Principais conceitos do OpenTelemetry
+aliases: [concepts/overview]
```

Comment:
I think this alias config can be removed.
Could you please confirm that, @chalin?

Reply:
@edsoncelio the docs in JA and ZH have the aliases. I copied and pasted rsrsrsrsrsrs

Reply:
Yes, generally speaking, such aliases, which pre-date the creation of the Portuguese translation, can be dropped.

Reply:
FYI, I've removed the `aliases` field.

Reply:
Thanks 👍